config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml

```diff
@@ -426,7 +426,7 @@ presubmits:
           limits:
             cpu: "2"
             memory: "6Gi"
-  - name: pull-kueue-test-scheduling-perf-main
+  - name: pull-kueue-test-perf-scheduling-main
```

Comment:
Actually, I would suggest to revert this because "scheduling-perf" is like a word combo here. Underneath it is using integration tests, so we could maybe rename as  `pull-kueue-test-integration-perf-scheduling-main`,
WDYT @tenzen-y ?

Reply:
This is actually `scheduling-perf` or `scheduler-perf`. So, +1 on reverting this line.
Indeed, this has a similar mechanism as k/k scheduler-perf: https://github.com/kubernetes/kubernetes/tree/e62ce1c9db2dadf225f37fe3dc943b64dc251950/test/integration/scheduler_perf

Reply:
Yeah, you're understanding correct

Reply:
I've reverted **all** `pull-kueue-test-perf-scheduling-` to `pull-kueue-test-scheduling-perf-` since they all call the same make target